pages/common/atom.md

```diff
@@ -0,0 +1,11 @@
+# atom
+
+> A hackable text editor for the 21st century.
```

Comment:
I still think it would be nice to highlight its main appeal in relation to other cross-platform text editors, but this isn't a strong opinion.